addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SstiScanRule.java

```diff
@@ -466,4 +476,9 @@ public List<Alert> getExampleAlerts() {
     public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
+
+    // For testing purposes only
+    static void setTamplateFormats(List<TemplateFormat> templates) {
```

Comment:
@kingthorin Did you mean `setTemplateFormats` for the function name?

Reply:
hahaha yeah, thanks. Will tweak.